libnetwork/store_linux_test.go

```diff
@@ -20,9 +21,14 @@ func TestBoltdbBackend(t *testing.T) {
 }
 
 func TestNoPersist(t *testing.T) {
-	testController, err := New(OptionBoltdbWithRandomDBFile(t))
+	dbFile := filepath.Join(t.TempDir(), "bolt.db")
+	testController, err := New(func(c *config.Config) {
+		c.Scope.Client.Provider = "boltdb"
+		c.Scope.Client.Address = dbFile
+		c.Scope.Client.Config = &store.Config{Bucket: "testBackend"}
+	})
```

Comment:
Assign the option closure to a variable so it can be reused on line 46.

Reply:
Ah, yes, makes sense; will do